### PR TITLE
Added S3_BUCKET to node_config

### PR DESCRIPTION
**Pull request by hem-brahmbhatt**
- Adds the config for the S3 bucket to NODE_CONFIG.
- Sets the application to production mode, so all features are enabled, and the application can use real APIs rather than local mocks.

### CODE REVIEW

**Review by diversemix — APPROVED** [review]
lgtm

File: docker-compose.yml [hunk 1]
```diff
@@ -34,7 +34,15 @@ services:
                 "auth-orcid": {
                     "clientID": "${ORCID_CLIENT_ID}",
                     "clientSecret": "${ORCID_CLIENT_SECRET}"
+                },
+                "meca": {
+                  "s3": {
+                    "params": {
+                      "Bucket": "${S3_BUCKET}",
```
Comment (giorgiosironi) [inline docker-compose.yml]: why is this Uppercase?

Comment (giorgiosironi) [reply docker-compose.yml]: indentation also seems to be off wrt the rest of the JSON dictionary

Comment (giorgiosironi) [reply docker-compose.yml]: usually `,` at the end of the last value is supported in JavaScript literals, but isn't in JSON; is that a problem?

**Comment by giorgiosironi** [comment]
Build seems stuck on `docker wait xpub_bootstrap_1` for the last 7 minutes.

File: docker-compose.yml (hunk 1 above)
Comment (hem-brahmbhatt) [reply docker-compose.yml]: `aws-sdk` likes it like that :(

**Comment by giorgiosironi** [comment]
Killed the build at https://alfred.elifesciences.org/blue/organizations/jenkins/pull-requests-projects%2Felife-xpub-deployment/detail/PR-9/1/pipeline
Does this work as expected locally?

**Review by giorgiosironi — APPROVED** [review]

**Pull request merged**